docs/src/admin-guide/ce/configuration.md

```diff
@@ -27,3 +29,91 @@ The following config assigns the role `admin` to all external users and this is
 5. Click save
 
 once steps above are followed a new button appears in the login form to start the authentication process for users coming from other organizations.
+
+## Datasets Importation
+
+::: tip
+**Importing** does not create explicit copies or duplication of files, only database entries and symbolic links are created.
+:::
+
+This section outlines the procedure for importing datasets, which are formatted according to the [BP Dataset File Structure Standard](https://bp.nbis.se/datasets/submission/preparation-guide.html).
+
+### Prerequisites
+
+- [Cytomine Python client](https://github.com/cytomine/cytomine/tree/main/clients/python) (v3.1.0+ recommended)
+- [sdafs](https://github.com/NBISweden/sdafs) (optional)
+
+::: tip
+`sdafs` is optional and only needed if you do not have the datasets on your filesystem.
+
+The installation guide is available in the [sdafs repository](https://github.com/NBISweden/sdafs?tab=readme-ov-file#sdafs).
+:::
+
+- The structure of the imported directory should follows:
+
+```
+root
+|--- {DATASET_UUID}
+|    |--- DATASET_{IDENTIFIER}
+|         |--- METADATA+
```

Comment:
what's this trailing `+` ?

Reply:
<img width="1227" height="582" alt="Screenshot from 2025-09-18 15-37-37" src="https://github.com/user-attachments/assets/ec1d1abf-800a-4143-869c-b9c8e5929ab8" />

This is to say that it is optional but since * is already there for another purpose I substitute it by +, but we can get rid of it